src/api/CustomUI.ts

```diff
@@ -160,9 +178,12 @@ export class CustomUI {
   private getHTML(panel: vscode.WebviewPanel, title: string) {
     const submitButton = this.fields.find(field => field.type === `submit`) || { id: `` };
 
-    const notInputFields = [`submit`, `buttons`, `tree`, `hr`, `paragraph`, `tabs`];
+    const notInputFields = [`submit`, `buttons`, `tree`, `hr`, `paragraph`, `tabs`, `complexTabs`];
     const trees = this.fields.filter(field => field.type == `tree`);
 
+    const complexTabFields = this.fields.filter(field => field.type === `complexTabs`).map(tabs => tabs.complexTabItems?.map(tab => tab.fields));
+    const allFields = [...this.fields, ...complexTabFields.flat().flat()].filter(cField => cField) as Field[];
```

Comment:
`map` then `flat`...`flatMap`? 😁
```suggestion
    const complexTabFields = this.fields.filter(field => field.type === `complexTabs`).flatMap(tabs => tabs.complexTabItems?.flatMap(tab => tab.fields) || []);
    const allFields = [...this.fields, ...complexTabFields];
```

Reply:
Fixed with `flat(2)` :)